trip_app/trips/models.py

```diff
@@ -4,8 +4,14 @@
 
 
 class Trip(models.Model):
-    driver = models.ForeignKey(settings.AUTH_USER_MODEL, related_name="suggested_trips", on_delete=models.CASCADE)
-    passengers = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name="booked_trips")
+    driver = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        related_name="suggested_trips",
+        on_delete=models.CASCADE,
+    )
+    passengers = models.ManyToManyField(
+        settings.AUTH_USER_MODEL, related_name="booked_trips"
+    )
     dep_time = models.DateTimeField()
     start_point = geo_models.PointField()
     dest_point = geo_models.PointField()
```

Comment:
Its better to use `decimal` type for any price/currency/money values. This type is more precise on calculations with floating points, so i suggest you to change this.

Reply:
Good point! Changed this field to decimal type. Thanks.